README.md

```diff
@@ -19,13 +19,36 @@ A library for rendering project templates.
 
 ![Sample output](https://github.com/pykong/copier/raw/master/img/copier-output.png)
 
-## How to install
+- [Installation](#installation)
+- [Quick usage](#quick-usage)
+- [Creating a template](#creating-a-template)
+  - [The `copier.yml` file](#the-copieryml-file)
+    - [Prompt the user for information](#prompt-the-user-for-information)
+      - [Advanced prompt formatting](#advanced-prompt-formatting)
+      - [Prompt templating](#prompt-templating)
+    - [Special options](#special-options)
+      - [Arguments defaults](#arguments-defaults)
+      - [Patterns syntax](#patterns-syntax)
+        - [Examples for pattern matching](#examples-for-pattern-matching)
+    - [Include other yaml files](#include-other-yaml-files)
+  - [The `.copier-answers.yml` file](#the-copier-answersyml-file)
+  - [Template helpers](#template-helpers)
+    - [Builtin variables/functions](#builtin-variablesfunctions)
+    - [Builtin filters](#builtin-filters)
+- [Generating a project](#generating-a-project)
+- [Updating a project](#updating-a-project)
+- [API](#api)
+  - [copier.copy()](#copiercopy)
+- [Comparison with other project generators](#comparison-with-other-project-generators)
+- [Credits](#credits)
```

Comment:
We don't want to maintain this manually. Replace it with:

```
<details>
<!-- prettier-ignore-start -->
<!-- START doctoc -->
<!-- END doctoc -->
<!-- prettier-ignore-end -->
</details>
```

Then, add to `.pre-commit-config.yaml` [this configuration](https://github.com/Tecnativa/doodba-copier-template/blob/9d7525f787019797306a10118b3da1368b0776a2/.pre-commit-config.yaml#L54-L62) and run it.

Reply:
It's updating CHANGELOG.md and CONTRIBUTING.md as well, how do I tell it not to update those?

Reply:
Well, maybe the best would be to add the same boilerplate and have TOCs there too.

Reply:
Sure why not then :slightly_smiling_face:

Reply:
OK done